root/programs/MultiPurposeAuthSite/MultiPurposeAuthSite/Controllers/ManageController.cs

```diff
@@ -0,0 +1,2440 @@
+        /// <returns>ActionResultを非同期に返す</returns>
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> SetPassword(ManageSetPasswordViewModel model)
+        {
+            // ManageSetPasswordViewModelの検証
+            if (ModelState.IsValid)
+            {
+                // ManageSetPasswordViewModelの検証に成功
+
+                // パスワード設定
+                IdentityResult result = await UserManager.AddPasswordAsync(User.Identity.GetUserId(), model.NewPassword);
+
+                // 結果の確認
+                if (result.Succeeded)
+                {
+                    // 成功
+
+                    // 再ログイン
+                    await this.ReSignInAsync();
+
+                    // イベント・ログ出力
+                    ApplicationUser user = await UserManager.FindByIdAsync(User.Identity.GetUserId());
+                    Logging.MyOperationTrace(string.Format("{0}({1}) did set own local password.", user.Id, user.UserName));
+
+                    // Index - SetPasswordSuccess
+                    return RedirectToAction("Index", new { Message = EnumManageMessageId.SetPasswordSuccess });
+                }
+                else
+                {
+                    // 失敗
+                    AddErrors(result);
+                }
+            }
+            else
+            {
+                // ManageSetPasswordViewModelの検証に失敗
+            }
+
+            // 再表示
+            return View(model);
+        }
+
+        #endregion
+
+        #region Update
+
+        /// <summary>
+        /// パスワード変更画面（初期表示）
+        /// GET: /Manage/ChangePassword
+        /// </summary>
+        /// <returns>ActionResult</returns>
+        [HttpGet]
+        public ActionResult ChangePassword()
+        {
+            return View();
+        }
+
+        /// <summary>
+        /// パスワード変更画面（パスワード変更）
+        /// POST: /Manage/ChangePassword
+        /// </summary>
+        /// <param name="model">ManageChangePasswordViewModel</param>
+        /// <returns>ActionResultを非同期に返す</returns>
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> ChangePassword(ManageChangePasswordViewModel model)
+        {
+            // ManageChangePasswordViewModelの検証
+            if (ModelState.IsValid)
+            {
+                // ManageChangePasswordViewModelの検証に成功
+
+                // パスワード変更
+                IdentityResult result = await UserManager.ChangePasswordAsync(User.Identity.GetUserId(), model.OldPassword, model.NewPassword);
+
+                // パスワードの変更結果の確認
+                if (result.Succeeded)
+                {
+                    // 成功
+
+                    // 再ログイン
+                    await this.ReSignInAsync();
+
+                    // イベント・ログ出力
+                    ApplicationUser user = await UserManager.FindByIdAsync(User.Identity.GetUserId());
+                    Logging.MyOperationTrace(string.Format("{0}({1}) did change own password.", user.Id, user.UserName));
```

Comment:
@daisukenishino 

In this sentence, account name is a subject, and password is an object.
So, I think that the phrase "*has changed*" is right.

If using the phrase "*has been changed*", password should be a subject.

```txt
Password of {0}({1}) has been changed.
```